### ISSUE TITLE
cocopp_commands.tex incomplete

### ISSUE THREAD

**Issue by brockho** [numbbo/coco#1265]
When running the postprocessing 2.0 (in the current `feature-refalg` branch) and specifying the output folder not directly where the postprocessing is run (e.g. `-o subfolder/ppdata`), I get an almost empty `cocopp_commands.tex` file that only has one entry:
```
\providecommand{\cocoversion}{\hspace{\textwidth}\scriptsize\sffamily{}\color{Gray}Data produced with COCO v1.2.1.3060}
```
As a result, the LaTeX template for a single algorithm does not compile. When using the standard output folder or specifying a folder directly where the code runs (e.g. `-o ppdataTEST`), everything works as normal.

Note that I did not test this functionality (yet) with other branches than the `feature-refalg` branch.

**Comment by dtusar** [comment]
I tried this and I the `cocopp_commands.tex` file of size 5 kB was created which should be fine. Do you have any idea in which case this works and in which it doesn't?

**Comment by brockho** [comment]
Okay, here a few more details about what worked and what not. All was run under Windows 7 without cygwin within the standard command line tool ("`cmd`"):

All this worked:
* `> python -m cocopp ALPS_hornby_noiseless.tgz`
* `> mkdir ppdatanew` with `> python -m cocopp -o ppdatanew ALPS_hornby_noiseless.tgz`
* `> mkdir ppdatanew\test` with `> python -m cocopp -o ppdatanew/test ALPS_hornby_noiseless.tgz`

But interestingly, when I changed the `/` into the more Windows-like `\`, it did *not* work anymore:
```
> mkdir ppdatanew\test2
> python -m cocopp -o ppdatanew\test2 ALPS_hornby_noiseless.tgz`
```
What was actually happening in the latter case was that two `cocopp_commands.tex` were created: one in the correct, expected `ppdatanew\test2\` folder and one with all the missing commands in `ppdatanew\`! The reason was this line in `rungeneric1.py`:
```
latex_commands_file = os.path.join(outputdir.split(os.sep)[0], 'cocopp_commands.tex')
```
I suggested a fix in the `feature-refalg branch` which already passed the CI tests such that I consider this issue as closed.
